src/cli/cli.cpp

```diff
@@ -551,6 +571,36 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
 }
 
 #if OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE
+void Interpreter::OutputBorderAgentCounters(const otBorderAgentCounters &aCounters)
+{
+    Uint64StringBuffer u64StringBuffer;
```

Comment:
Using `uint32_t` or `uint16` will help simplify this code as well (no need to use `Uint64ToString()`).

Reply:
Done